server/ec2spotmanager/models.py

```diff
@@ -34,71 +30,60 @@ def get_available_name(self, name, max_length=None):
 
 
 class PoolConfiguration(models.Model):
-    parent = models.ForeignKey("self", blank=True, null=True)
+    parent = models.ForeignKey('self', blank=True, null=True)
     name = models.CharField(max_length=255, blank=False)
     size = models.IntegerField(default=1, blank=True, null=True)
     cycle_interval = models.IntegerField(default=86400, blank=True, null=True)
+    max_price = models.DecimalField(max_digits=12, decimal_places=6, blank=True, null=True)
+    allowed_regions = models.CharField(max_length=1023, blank=True, null=True)
```

Comment:
We'll have to rename this back to `ec2_allowed_regions` when we add other providers.